startDevelopmentStack.sh

```diff
@@ -29,3 +29,5 @@ docker compose --profile development up -d --build
 
 #start the backend
 ./gradlew dataland-backend:bootRun --args='--spring.profiles.active=development' --no-daemon --stacktrace
+
+sleep 10000
```

Comment:
why?

Reply:
Was my mistake, i comitted my debugging statements. Sorry for that

Reply:
ok